src/ott/solvers/quadratic/lower_bound.py

```diff
@@ -49,31 +42,42 @@ def __init__(
       **kwargs: Any,
   ):
     self.epsilon = epsilon
-    self.univariate_solver = univariate.UnivariateSolver(**kwargs)
 
   def __call__(
       self,
       prob: quadratic_problem.QuadraticProblem,
-      **kwargs: Any,
+      rng: Optional[jax.Array] = None,
+      kwargs_univsolver: Optional[Any] = None,
+      epsilon: Optional[float] = None,
```

Comment:
I'd put `epsilon` above `rng`.

Reply:
thanks! also updated the `Returns` field